### DISCUSSION TITLE
[Feature] Export all customization, users & crm settings

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#13972] [Wishlist]
Hi guys,
is it possible to export all crm customizations and import them into a new fresh crm installation?

or do i  have to manually create all users again, profile n roles , custom fields in all modules that i made..
is their an easy way ?

**Comment by apcloic** [comment]
@rioshaz
Hi, 
As this is a request and not a bug, I think you should post your question here :
https://github.com/YetiForceCompany/YetiForceCRM/discussions
Regards,

**Comment by rioshaz** [comment]
i have posted this as a "feature" not a bug sir

also my crm screen goes blank.. DOES NOT  load any data/records, system setting icon disappears

![image](https://user-images.githubusercontent.com/54880467/103149918-6333c080-4794-11eb-904b-e8f8b5b4ee59.png)


and in chrome inspect console i see this 

![image](https://user-images.githubusercontent.com/54880467/103149902-2cf64100-4794-11eb-90d6-c06e39f21a66.png)

can you help...

**Reply by mariuszkrzaczkowski** [reply]
Report each bug on separate issues

**Comment by apcloic** [comment]
@rioshaz 
Issues section is now only for bugs.
I think you've missed the following announcement
#13908
